spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -7,14 +7,17 @@
 This section covers the changes made from version 3.1 to version 3.2.
 For changes in earlier version, see xref:appendix/change-history.adoc[Change History].
 
+[[x32-kafka-client-version]]
+=== Kafka Client Version
+
+This version requires 3.7.0 `kafka-clients`.
```

Comment:
Don’t we have anywhere else in docs the mentioning for Kafka Client compatibility ?

Reply:
The compatibility matrix is on the project page, which we need to edit later, maybe once this is merged.